README

```diff
@@ -1,44 +1,35 @@
 # rsv-gwms-tester
-Additional probes and configuration. One probe that activates several vanilla probes per sites.
 
-Requirements:
-
-1. Valid Host cert
-
-
-In order to install:
-
-1. Install OSG Repos as:
-
-https://twiki.grid.iu.edu/bin/view/Documentation/Release3/YumRepositories
-
-2. yum install rsv-gwms-tester
-
-
-Configuration:
-
-1. Create file /etc/condor-cron/config.d/98_location.config with contents:
-
-CONDOR_HOST = your_glidein_pool_central_manager
-# Don't try the IP address macro of condor-cron it defaults to the private IP
-NETWORK_INTERFACE = this_host_ip
-
-2.  Create file /etc/condor-cron/config.d/90_gwms_dns.config
-
-# Comment: central manager your_central_manager_hostname
-# The following DN will map to condor                                                                                                                                            
-GSI_DAEMON_NAME=$(GSI_DAEMON_NAME),your_central_manager_DN
-
-3. Create file /etc/condor-cron/certs/condor_mapfile
-
-Add central manager DN to the list:
-
-like: 
-GSI "^\/DC\=org\/DC\=opensciencegrid\/O\=Open\ Science\ Grid\/OU\=Services\/CN\=osg\-ligo\-1\.t2\.ucsd\.edu$" condor
-
-
-To start:
-
-rsv-control --enable org.osg.local-gfactory-site-querying-local --host host_name
-
-rsv-control --on org.osg.local-gfactory-site-querying-local --host host_name
\ No newline at end of file
+Use Case
+========
+Rsv-gwms-tester is a package that is meant to do end-to-end regular testing of [GlideinWMS](https://glideinwms.fnal.gov/).
+More specifically it is mean to submit a set of HTCondor Vanila universe jobs to the set of supported grid sites of a Virtual Organization (VO).
+It can be thought of as a [SAM-light](https://twiki.cern.ch/twiki/bin/view/CMSPublic/CompOpsSAMTests) set of grid tests.
+
+Requirements
+============
+1. The Virtual Organization has already a working [Glideinwms Frontend](https://glideinwms.fnal.gov/doc.prd/frontend/index.html)
+1. The Virtual Organization creates a set of probes(jobs or capabilities it wants to test out of a site).
+1. A Rhel6/6 machine that can install rsv as root.
```

Comment:
```suggestion
1. An EL6 machine that can install rsv as root.
```

Why the EL6 requirement?

Reply:
Actually it just needs a RHEL 6/7 machine.